src/backend/player_command.hpp

```diff
@@ -32,6 +32,21 @@ namespace big
 			return {0};
 		};
 
+		virtual std::optional<std::vector<std::string>> get_argument_suggestions(int arg) override
+		{
+			if (arg == 1) // First argument of all player commands is the player name
+			{
+				std::vector<std::string> suggestions;
+				for (auto& [_, player] : g_player_service->players())
```

Comment:
If you're not making use of the key then consider reworking the for loop to make use of `std::ranges::views::values`.

```cpp
for (const auto& session_type_string : m_session_types | std::ranges::views::values)
{

}
```